WebSite/Site.Master

```diff
@@ -33,6 +33,7 @@
                   <li><a href="csrf.aspx"> CSRF</a></li>
                   <li><a href="Deserialization.aspx"> Deserializattion</a></li>
                   <li><a href="idor.aspx"> IDOR</a></li>
+                  <li><a href="Misc.aspx"> MISC</a></li>
```

Comment:
IS 'MISC' the best name for this? When people see 'MISC' on the menu, they're not going to know what it is referring to.

Reply:
What would you suggest? Idea was to have a page that contains miscellaneous functions.

Reply:
Do you feel that the category you just added doesn't deserve its own place on the top menu? If not, that's cool - I'd suggest 'Other' rather than 'Misc', in that case.